mmcv/ops/contour_expand.py

```diff
@@ -42,7 +45,6 @@ def contour_expand(kernel_mask, internal_kernel_label, min_kernel_area,
                 internal_kernel_label,
                 min_kernel_area=min_kernel_area,
                 kernel_num=kernel_num)
-            label = label.tolist()
```

Comment:
This variable `label` is Tensor here.

label = label.tolist()  # type: ignore

Reply:
It was an error from mmcv master.

You can merge the upstream master to fix the CI with the following steps.

```bash
git remote add open-mmlab git@github.com:open-mmlab/mmcv.git
git pull open-mmlab master
git push origin ytzhao/type_hints_runner_builder
```